releasenotes/notes/0.23/prepare-0.23.0-release-0d954c91143cf9a4.yaml

```diff
@@ -0,0 +1,124 @@
+---
+prelude: |
+  Qiskit Terra 0.23.0 is a major feature release that includes
+  a multitude of new features and bugfixes. The highlights for this release
+  are:
+
+    * Support for importing `OpenQASM3` programs and creating :class:`.QuantumCircuit` objects
+      from the input program via two new functions :func:`qiskit.qasm3.load` and
+      :func:`qiskit.qasm3.loads`.
```

Comment:
Should we mention that you need an optional (`pip install "qiskit-terra[qasm3-import]"`) for this?

Reply:
I figured it was ok to omit here because we mention it in the feature note. But, it doesn't hurt to add it here too I guess.